js/app.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+// BookConstructor
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.bookId = Math.random().toFixed(1);
+  }
+}
```

Comment:
- [OPTIONAL] Great job adding `book ID` 💯 However,  Using `Math.random().toFixed(1);` it's not bad, But It does not guarantee the uniqueness of the book ⬇

![image](https://user-images.githubusercontent.com/45231276/204512915-d7ec6a49-0f94-4b9a-811f-4eb9404876ce.png)

So I suggest using `new Date().getTime();` instead
> For Example 💡
```js
 this.bookId = new Date().getTime();
```

Reply:
Thank you so much for your reviews, Sir.